src/content/docs/usage/Node Graph/Tutorials/your-first-graph.mdx

```diff
@@ -11,69 +11,94 @@ import { Steps } from '@astrojs/starlight/components';
 import { Image } from 'astro:assets';
 import ctxMenuImage from './assets/ctx-menu.png';
 import { Quiz, QuizOption } from 'starlight-videos/components'
+import searchNodes from '.assets/search-for-nodes';
 
-First of all, congratulations on deciding to learn about Node Graphs in PixiEditor! It's a fascinating and powerful system, you won't regret it.
+Welcome to Nodes.🎉 The Node system is a powerful way to build and control animations in a non-destructive, flexible workflow.    
```

Comment:
This is indirectly misleading. Node system is not only for animations, but it's a core component of whole PixiEditor. It allows for:

- Creating procedural artworks
- Editing images
- Creating animations
- Building custom workspaces

and in the future custom brushes